### PR TITLE
Proximity analysis

### PR DESCRIPTION
**Pull request by HypnoMama**
Add api calls to retrieve data for nearby resources within 50km, display table of nearby resources with nearest resource, display color-coded markers for resources within 50km.

### CODE REVIEW

File: cit3.0-web/src/components/MapContainer/MapContainer.js [hunk 1]
```diff
@@ -37,19 +46,38 @@ export default function MapContainer() {
   };
 
   return (
-    <div className="d-flex flex-column justify-content-between">
-      <Input input={{ ...input }} onChange={setAddress} />
+    <div className="d-flex flex-column justify-content-between align-items-center w-100">
+      <div className="w-100 mb-2 pt-2">
```
Comment (BrendanBeachBC) [inline cit3.0-web/src/components/MapContainer/MapContainer.js:50]: Same question as above for inline CSS

File: cit3.0-web/src/components/MapContainer/MapContainer.js [hunk 2]
```diff
@@ -37,19 +46,38 @@ export default function MapContainer() {
...
   );
 }
+
+MapContainer.defaultPropTypes = {
+  nearbyResources: null,
+};
+
+MapContainer.propTypes = {
+  // eslint-disable-next-line react/require-default-props
+  nearbyResources: PropTypes.shape({
```
Comment (BrendanBeachBC) [inline cit3.0-web/src/components/MapContainer/MapContainer.js:78]: Should we have a default prop for this, or maybe make it a required prop?

File: cit3.0-web/src/helpers/resourceCalls.js [hunk 3]
```diff
@@ -0,0 +1,97 @@
+/* eslint-disable prettier/prettier */
```
Comment (BrendanBeachBC) [inline cit3.0-web/src/helpers/resourceCalls.js:1]: Is Prettier causing an issue in this file?

**Review by BrendanBeach — COMMENTED** [review]
Looks pretty good, just a couple of general questions

**Review by BrendanBeach — APPROVED** [review]

**Pull request merged**